### PR TITLE
Add DatePicker to Timetable and Attendance modules.

### PR DESCRIPTION
**Pull request by v0idzz**
This PR enables the user to pick a date in Timetable and Attendance modules using DatePickerDialog.

### CODE REVIEW

**Comment by freshstudio** [comment]
PR zamyka issue #495 czy dalej myślimy jak polepszyć nawigację między datami?

File: app/src/main/java/io/github/wulkanowy/ui/modules/attendance/AttendanceFragment.kt [hunk 1]
```diff
@@ -141,6 +145,18 @@ class AttendanceFragment : BaseFragment(), AttendanceView, MainView.MainChildVie
         (activity as? MainActivity)?.showDialogFragment(AttendanceDialog.newInstance(lesson))
     }
 
+    override fun showDatePickerDialog(currentDate: LocalDate, baseDate: LocalDate) {
```
Comment (mklkj) [inline app/src/main/java/io/github/wulkanowy/ui/modules/attendance/AttendanceFragment.kt]: Remove unused `baseDate` param

File: app/src/main/java/io/github/wulkanowy/ui/modules/attendance/AttendanceFragment.kt [hunk 2]
```diff
@@ -71,6 +74,7 @@ class AttendanceFragment : BaseFragment(), AttendanceView, MainView.MainChildVie
 
         attendanceSwipe.setOnRefreshListener(presenter::onSwipeRefresh)
         attendancePreviousButton.setOnClickListener { presenter.onPreviousDay() }
+        attendanceNavDate.setOnClickListener {presenter.onPickDate() }
```
Comment (mklkj) [inline app/src/main/java/io/github/wulkanowy/ui/modules/attendance/AttendanceFragment.kt]: Please follow our code style rules - they can be applied automatically by pressing appropriate shortcut https://developer.android.com/studio/intro#style_and_formatting

File: app/src/main/java/io/github/wulkanowy/ui/modules/timetable/TimetableFragment.kt [hunk 3]
```diff
@@ -144,6 +147,18 @@ class TimetableFragment : BaseFragment(), TimetableView, MainView.MainChildView,
         (activity as? MainActivity)?.showDialogFragment(TimetableDialog.newInstance(lesson))
     }
 
+    override fun showDatePickerDialog(currentDate: LocalDate, baseDate: LocalDate) {
+        val dateSetListener = com.wdullaer.materialdatetimepicker.date.DatePickerDialog.OnDateSetListener { _, year, month, dayOfMonth ->
```
Comment (mklkj) [inline app/src/main/java/io/github/wulkanowy/ui/modules/timetable/TimetableFragment.kt]: Why you use fully-qualified name? Use import instead

File: app/src/main/java/io/github/wulkanowy/utils/SchooldaysRangeLimiter.kt [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+    companion object CREATOR : Parcelable.Creator<SchooldaysRangeLimiter> {
+        override fun createFromParcel(parcel: Parcel): SchooldaysRangeLimiter {
+            return SchooldaysRangeLimiter(parcel)
+        }
+
+        override fun newArray(size: Int): Array<SchooldaysRangeLimiter?> {
+            return arrayOfNulls(size)
+        }
+    }
+}
```
Comment (mklkj) [inline app/src/main/java/io/github/wulkanowy/utils/SchooldaysRangeLimiter.kt]: Please add new line at the end of file

**Comment by freshstudio** [comment]
All of the issues you listed have been addressed.

**Comment by mklkj** [comment]
Maybe set layout version to 1
`datePickerDialog.version = DatePickerDialog.Version.VERSION_1`
to avoid this (black arrows in dark theme) https://imgur.com/a/Nk1VsQw

**Pull request merged**